python/ray/rllib/ppo/filter.py

```diff
@@ -92,16 +92,21 @@ def __init__(self, shape, demean=True, destd=True, clip=10.0):
         self.destd = destd
         self.clip = clip
         self.rs = RunningStat(shape)
+        self.buffer = RunningStat(shape)
```

Comment:
I'd suggest adding a doc explaining the purpose of `self.buffer` and how it is used in `self.update`. That's the main thing that could be confusing.